hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenClaimAirdropHandler.java

```diff
@@ -44,10 +55,46 @@ public TokenClaimAirdropHandler() {
     }
 
     @Override
-    public void preHandle(@NonNull PreHandleContext context) throws PreCheckException {}
+    public void preHandle(@NonNull PreHandleContext context) throws PreCheckException {
+        requireNonNull(context);
+        final var op = context.body().tokenClaimAirdrop();
+        requireNonNull(op);
+        final var pendingAirdrops = op.pendingAirdrops();
+
+        final var accountStore = context.createStore(ReadableAccountStore.class);
+
+        for (final var pendingAirdrop : pendingAirdrops) {
+            AccountID receiverId = pendingAirdrop.receiverIdOrThrow();
+            Account account = accountStore.getAccountById(receiverId);
+            requireNonNull(account);
+            Key key = account.key();
+            requireNonNull(key);
```

Comment:
Instead of `requireNonNull` , we should use `validateTruePrecheck`

Reply:
Changed 👍 
But why do we need this change? Is it because we want to throw `PreCheckException` instead of `NullPointerException`? If so - do we want to change the rest of the `requireNonNull` from the same method?